storagenode/bandwidth/db_test.go

```diff
@@ -75,35 +67,227 @@ func TestDB(t *testing.T) {
 			require.NoError(t, err)
 		}
 
-		// test summarizing
-		usage, err = bandwidthdb.Summary(ctx, now.Add(-10*time.Hour), now.Add(10*time.Hour))
-		require.NoError(t, err)
-		require.Equal(t, expectedUsageTotal, usage)
-
 		expectedUsageBySatellite := map[storj.NodeID]*bandwidth.Usage{
 			satellite0: expectedUsage,
 			satellite1: expectedUsage,
 		}
-		usageBySatellite, err = bandwidthdb.SummaryBySatellite(ctx, now.Add(-10*time.Hour), now.Add(10*time.Hour))
-		require.NoError(t, err)
-		require.Equal(t, expectedUsageBySatellite, usageBySatellite)
 
-		// only range capturing second satellite
-		usage, err = bandwidthdb.Summary(ctx, now.Add(time.Hour), now.Add(10*time.Hour))
-		require.NoError(t, err)
-		require.Equal(t, expectedUsage, usage)
+		// test summarizing
+		t.Run("test total summary", func(t *testing.T) {
+			usage, err := bandwidthdb.Summary(ctx, now.Add(-10*time.Hour), now.Add(10*time.Hour))
+			require.NoError(t, err)
+			require.Equal(t, expectedUsageTotal, usage)
 
-		cachedBandwidthUsage, err := bandwidthdb.MonthSummary(ctx)
-		require.NoError(t, err)
-		require.Equal(t, expectedUsageTotal.Total(), cachedBandwidthUsage)
+			// only range capturing second satellite
+			usage, err = bandwidthdb.Summary(ctx, now.Add(time.Hour), now.Add(10*time.Hour))
+			require.NoError(t, err)
+			require.Equal(t, expectedUsage, usage)
+		})
+
+		t.Run("test summary by satellite", func(t *testing.T) {
+			usageBySatellite, err := bandwidthdb.SummaryBySatellite(ctx, now.Add(-10*time.Hour), now.Add(10*time.Hour))
+			require.NoError(t, err)
+			require.Equal(t, expectedUsageBySatellite, usageBySatellite)
+
+			// only range capturing second satellite
+			expectedUsageBySatellite := map[storj.NodeID]*bandwidth.Usage{
+				satellite1: expectedUsage,
+			}
+
+			usageBySatellite, err = bandwidthdb.SummaryBySatellite(ctx, now.Add(time.Hour), now.Add(10*time.Hour))
+			require.NoError(t, err)
+			require.Equal(t, expectedUsageBySatellite, usageBySatellite)
+		})
+
+		t.Run("test satellite summary", func(t *testing.T) {
+			usage, err := bandwidthdb.SatelliteSummary(ctx, satellite0, time.Time{}, now)
+			require.NoError(t, err)
+			require.Equal(t, expectedUsageBySatellite[satellite0], usage)
+
+			usage, err = bandwidthdb.SatelliteSummary(ctx, satellite1, time.Time{}, now.Add(10*time.Hour))
+			require.NoError(t, err)
+			require.Equal(t, expectedUsageBySatellite[satellite1], usage)
+		})
+
+		t.Run("test cached bandwidth usage", func(t *testing.T) {
+			cachedBandwidthUsage, err := bandwidthdb.MonthSummary(ctx)
+			require.NoError(t, err)
+			require.Equal(t, expectedUsageTotal.Total(), cachedBandwidthUsage)
+		})
+	})
+}
+
+func TestEmptyBandwidthDB(t *testing.T) {
+	storagenodedbtest.Run(t, func(t *testing.T, db storagenode.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		bandwidthdb := db.Bandwidth()
+
+		now := time.Now().UTC()
```

Comment:
For tests here, do not use `UTC()`. I think it's an easy mistake from the caller side to make, so it's probably better for the database internally handle it.